services/productCategoryService.js

```diff
@@ -0,0 +1,18 @@
+const productCategoryModel = require('../models/productCategoryModel');
+
+const deleteProductCategory = async (req, res, next) => {
+  const productCategory = {
+    ...req.body, // product_id and category_id should be passed in body
+  };
+
+  try {
+    await productCategoryModel.deleteProductCategory(productCategory);
+  } catch (e) {
+    console.log(e);
+    next(e);
+  }
+};
+
+module.exports = {
+  deleteProductCategory,
```

Comment:
We'll probably want `addProductCategory` too, but not a blocker at all

Reply:
thanks, done!